package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@oclif/core",
   "description": "base library for oclif CLIs",
   "version": "1.0.3",
-  "author": "Jeff Dickey @jdxcode",
+  "author": "Salesforce",
```

Comment:
Maybe `Salesforce CLI Team`?
Doesn't really matter tho.

Reply:
I just went with what's in sfdx and sf

https://github.com/salesforcecli/cli/blob/main/package.json#L5
https://github.com/salesforcecli/sfdx-cli/blob/main/package.json#L5